src/styles/foundation/_focus-ring.scss

```diff
@@ -27,6 +29,21 @@
   }
 }
 
+@mixin focus-ring-animated($size: 'base', $show: 0) {
```

Comment:
My thinking was this would offer the same features as the non-animated focus ring would.

Reply:
If we make don't expose this as part of our public API then we're free to only add it when we need it.

Reply:
The focus ring will have to be public in some way, whether that ends up coming from polaris-react or polaris-tokens (although a mixin in tokens is questionable). Teams building custom components with focus styles will want to use the same focus ring.

Reply:
Should the animation be added to all rings?

Reply:
I'd like to see this refactored so that all components which use the focus ring can consume the same mixin.

I'd also like to see some indication of whether this mixin works for `Button` and `TextField` as well as `Checkbox` and `Radio`.